src/common/constants.ts

```diff
@@ -0,0 +1,20 @@
+import { AresRpgEngine } from "aresrpg-engine";
+import { Color } from "three";
+
+enum EVoxelType {
+    ROCK,
+    GRASS,
+    SNOW,
+    WATER,
+    SAND,
+}
+
+const VOXEL_TYPE_COLORS: Record<EVoxelType, AresRpgEngine.IVoxelMaterial> = [
+    { color: new Color("#ABABAB") },
+    { color: new Color("#00B920") },
+    { color: new Color("#E5E5E5") },
+    { color: new Color("#0055E2") },
+    { color: new Color("#DCBE28") },
+];
```

Comment:
I assume the less information we transfer from the CPU to the GPU the better ? it would depend on how the rendering engine is handling those but the terrain generator can supply IDs and it's up to another entity to define what they correspond to, Especially if they are mapped directly in shaders

Reply:
In fact at the end it doesn't change the way the engine is using it so no impact on GPU/CPU.
Most important is that 
- generator knows what are the different types of material it is supposed to generate (e.g. grass, rock, sand, ...)
- engine knows how specific voxel type should be rendered (e.g. grass as green, rock as gray, ....)
As the generator is the one who knows what types of material it will generate, I would think this is responsability of generator to tell the engine how it should be rendered but we can further discuss it with Jeremy to see what makes more sense.